roles/cinder-control/tasks/main.yml

```diff
@@ -7,10 +7,12 @@
     name: "{{ item.name }}"
     user: "{{ item.user }}"
     cmd: "{{ item.cmd }}"
+    envs: "{{ cinder.service.envs }}"
```

Comment:
Why are we trying to enable it for RHOSP? I thought the scope of the cinder volume encryption was for Ubuntu ONLY. Also, I don't think monolithic PRs are a good thing.